### PR TITLE
Update Guard.java

### PR DESCRIPTION
**Pull request by oliviaxjiang**

### CODE REVIEW

File: core/src/com/raccoon/mygame/models/Guard.java [hunk 1]
```diff
@@ -1,5 +1,68 @@
...
+
+    private Texture patrolTexture;
+
+    Guard(float x, float y, float width, float height, Texture texture){
+        position = new Vector2(0.0f, 0.0f);
+        velocity = new Vector2(0.0f, 0.0f);
+        this.width = width;
+        this.height = height;
+        patrolTexture = texture;
+        position = new Vector2(x, y);
```
Comment (liamgiraldo) [inline core/src/com/raccoon/mygame/models/Guard.java]: you say position = new Vector2() twice. You should only need the last one where you set it equal to x and y.

Comment (oliviaxjiang) [reply core/src/com/raccoon/mygame/models/Guard.java]: oops thanks for catching that, will make a commit to get rid of the last position definition

Comment (oliviaxjiang) [reply core/src/com/raccoon/mygame/models/Guard.java]: > you say position = new Vector2() twice. You should only need the last one where you set it equal to x and y.

just fixed, commited and pushed!

File: core/src/com/raccoon/mygame/models/Guard.java [hunk 2]
```diff
@@ -1,5 +1,67 @@
...
+
+    private float width;
+    private float height;
+
+    private Vector2 velocity;
+
+    private Texture patrolTexture;
+
+    Guard(float x, float y, float width, float height, Texture texture){
+        position = new Vector2(0.0f, 0.0f);
```
Comment (liamgiraldo) [inline core/src/com/raccoon/mygame/models/Guard.java]: shouldn't it be position = new Vector2(x,y)?

Comment (oliviaxjiang) [reply core/src/com/raccoon/mygame/models/Guard.java]: mmm true just changed and commited

**Pull request merged**